transmart-rest-api/src/main/groovy/org/transmartproject/rest/dataExport/ExportJobExecutor.groovy

```diff
@@ -0,0 +1,99 @@
+package org.transmartproject.rest.dataExport
+
+import grails.persistence.support.PersistenceContextInterceptor
+import grails.util.Holders
+
+import groovy.util.logging.Slf4j
+import org.apache.commons.lang.StringUtils
+import org.quartz.Job
+import org.quartz.JobExecutionContext
+import org.transmartproject.core.exceptions.UnexpectedResultException
+
+import java.util.zip.ZipOutputStream
+
+/**
+ * This class will encompass the job scheduled by Quartz.
+ */
+@Slf4j
+class ExportJobExecutor implements Job {
+
+    def ctx = Holders.grailsApplication.mainContext
+
+    RestDataExportService restDataExportService = ctx.restDataExportService
+    ExportAsyncJobService asyncJobService = ctx.exportAsyncJobService
+//    QuartzSpringScope quartzSpringScope = ctx.quartzSpringScope
+
+    final String tempFolderDirectory = Holders.config.com.recomdata.plugins.tempFolderDirectory
+
+    public void execute(JobExecutionContext jobExecutionContext) {
+        def userInContext = jobExecutionContext.jobDetail.jobDataMap['userInContext']
+        Map jobDataMap = jobExecutionContext.jobDetail.getJobDataMap()
+
+//        // put the user in context
+//        quartzSpringScope.currentUserBeanQuartzScope = userInContext
```

Comment:
This was a part of transmartApp async job executor, I am not sure why and if it is needed (also setting currentUserBeanQuartzScope). @JanKanis, @forus do you know, if something with similar to [QuartzSpringScope](https://github.com/thehyve/transmart-core/blob/master/transmartApp/src/main/groovy/org/transmart/spring/QuartzSpringScope.groovy) from tm-app should be added here?